test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDuplicateMovementReferenceNumberControllerSpec.scala

```diff
@@ -136,30 +141,60 @@ class EnterDuplicateMovementReferenceNumberControllerSpec
 
         val doc = Jsoup.parse(contentAsString(performAction()))
 
-        doc.select("h1").text should include(messageFromMessageKey("enter-duplicate-movement-reference-number.title"))
+        doc.select("h1").text should include(messageFromMessageKey(s"$key.title"))
+      }
+
+      "Fail if the same Entry Number is submitted and legacy journey is disabled" in {
+        featureSwitch.EntryNumber.disable()
+
+        val entryNumber     = EntryNumber("123456789A12345678")
```

Comment:
there is an Entry number generator now which we can use here

Reply:
It is something I asked @caoilte to do back then. I should be addressed by merging `master`.